documentation/using-hyperview/sensor-management/index.rst

```diff
@@ -0,0 +1,76 @@
+*****************
+Sensor Management
+*****************
+
+============
+Key concepts
+============
+
+Sensor categories
+-----------------
+Sensor data is one of the primary data and analytics pipelines in Hyperview. Sensors have to be associated with assets and have the following categories:
+
+1. Direct: these sensors are directly attached to assets. For example, the Total Output Power sensor is read from a rack PDU. Data collectors update direct sensors.
+2. Linked: these sensors are sourced from other assets. For example, a temperature sensor discovered on an environmental monitor may provide information on the temperature of a rack.
```

Comment:
"These sensors..." instead of "these sensors..."

Reply:
Thanks! Updated.